src/term.c

```diff
@@ -541,6 +541,10 @@ static int cmd_sig(PROGRAMMER * pgm, struct avrpart * p,
 static int cmd_quit(PROGRAMMER * pgm, struct avrpart * p,
 		    int argc, char * argv[])
```

Comment:
You need to replace all tabs with spaces

Reply:
These were in the original code, the patch didn’t touch it.

Reply:
> These were in the original code, the patch didn’t touch it.

This is OK. AVRDUDE has a multitude of styles, due to its history.
Unchanged code better remains unchanged, just new code should avoid TABs.